docs/security/trust/trust_key_mng.md

```diff
@@ -11,19 +11,34 @@ parent= "smn_content_trust"
 # Manage keys for content trust
 
 Trust for an image tag is managed through the use of keys. Docker's content
-trust makes use four different keys:
+trust makes use of five different types of keys:
```

Comment:
I've updated the content trust document to present a more simplistic view of the keys - I didn't want to eliminate it entirely, because the diagram is probably pretty useful, but it no longer has that table (although it still kind of has a list of classes of keys :|)